src/test/kotlin/uk/gov/justice/hmpps/offendersearch/controllers/OffenderSearchPhraseAPIIntegrationTest.kt

```diff
@@ -613,15 +666,21 @@ class OffenderSearchPhraseAPIIntegrationTest {
         .body("total", equalTo(0))
   }
 
-  private fun doSearch(phrase: String, matchAllTerms: Boolean = false): ValidatableResponse {
+  private fun doSearch(phrase: String, matchAllTerms: Boolean = false, size: Int? = null, page: Int? = null): ValidatableResponse {
+    val filter = SearchPhraseFilter(
```

Comment:
Can't find a beautiful Kotlin way of providing values when not null, so went for this ..